src/lambda_streams/Numbers.java

```diff
@@ -11,81 +11,248 @@ public static void main(String[] args) {
+                sum = sum - nums.get(i);
+            }
+            System.out.println(sum);
+            return sum;
+        };
+        LambdaMissVar.Compute(nums);
 
-        OR
+        Calulate2 LambdaDivideVar = (nums) -> {
+            long sum = 0;
+            for (int i = 0; i < nums.size(); i++) {
+                sum = sum / nums.get(i);
+            }
+            System.out.println(sum);
+            return sum;
+        };
+        LambdaDivideVar.Computer(nums);
 
-        lambdaAdd = () -> {};
-        added(lambdaAdd);
+        Calulate2 LambdaMultiVar = (nums) -> {
+            long sum = 0;
+            for (int i = 0; i < nums.size(); i++) {
+                sum = sum * nums.get(i);
+            }
+            System.out.println(sum);
+            return sum;
+        };
+        LambdaMultiVar.Computer(nums);
 
-        isOdd(() -> {});
+        PickNumber LambdaOddValue = (int i) -> {
+            if(nums.get(i) % 2 == 1){
+                System.out.println("This is True");
+                return true;
+            }
+            else {
+                System.out.println("This is false");
+                return false;
+            }
+        };
+        LambdaOddValue.Pick(3);
 
-        OR
+        PickNumber LambdaEvenValue = (int i) -> {
+            if(nums.get(i) % 2 == 1){
+                System.out.println("This is false");
+                return true;
+            }
+            else {
+                System.out.println("This is true");
+                return false;
+            }
+        };
+        LambdaEvenValue.Pick(3);
 
-        lambdaOdd = () -> {};
-        isOdd(lambdaOdd);
-        etc...
+        PickNumber LambdaPrimeValue = (int i) -> {
+            for (int x = 2; x < Math.sqrt(i); x++) {
+                if (i % x == 0) {
+                    System.out.println("False");
+                    return false;
+                }
+            }
+            System.out.println("True");
+            return true;
+        };
+        LambdaPrimeValue.Pick(3);
 
-        */
+        Calulate LambdaMaxValue = (nums) -> {
+            int max = 0;
+            for(int i = 0; i < nums.size(); i++){
+                if(nums.get(i) > max){
+                    max = nums.get(i);
+                }
+            }
+            System.out.println(max);
+            return max;
+        };
+        LambdaMaxValue.Compute(nums);
 
+        Calulate LambdaMinValue = (nums) -> {
+            int min = 0;
+            for(int i = 0; i < nums.size(); i++){
+                if(nums.get(i) < min){
+                    min = nums.get(i);
+                }
+            }
+            System.out.println(min);
+            return min;
+        };
+        LambdaMinValue.Compute(nums);
+
+        Comp LambdaComp = (int i, int j) -> {
+            if(nums.get(i) > nums.get(j)){
+                System.out.println(1);
+                return 1;
+            }
+            else if(nums.get(j) > nums.get(i)){
```

Comment:
When multiple 'If/Then' statements are comparing the same numbers, keep the numbers in order and change the operator to avoid confusion.